code/modules/surgery/organs/augments_internal.dm

```diff
@@ -291,7 +291,7 @@
 		owner.unEquip(holder, 1)//if he does, take it away.
 		holder.loc = null
 		out = 0
-		owner.visible_message("<span class='notice'>[holder] forcibly retracts into your arm.</span>")
+		owner << "<span class='warning'>[holder] forcibly retracts into your arm.</span>"
```

Comment:
This means the owner will get the message even if they're dead when the implant gets EMPed.